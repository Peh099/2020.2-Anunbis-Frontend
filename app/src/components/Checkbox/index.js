import React from 'react';
import { Checkmark, Container } from './styles';

export default function Checkbox({ text, name, register, checked, onChange }) {
  return (
    <Container>
      <input type="checkbox" name={name} ref={register} checked={checked} onChange={onChange}/>
      <Checkmark />
      {text}
    </Container>
  );
}