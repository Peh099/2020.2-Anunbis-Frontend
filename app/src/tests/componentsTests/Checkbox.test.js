import renderer from 'react-test-renderer';
import Checkbox from '../../components/Checkbox';

describe('Snapshot Checkbox component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Checkbox />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
