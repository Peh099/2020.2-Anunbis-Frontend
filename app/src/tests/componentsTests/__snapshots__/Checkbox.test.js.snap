// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Checkbox component matches the snapshot 1`] = `
<label
  className="sc-gtsrHT cVdwhi"
>
  <input
    type="checkbox"
  />
  <span
    className="sc-bdnxRM eHCeev"
  />
</label>
`;
